docs/examples/cookbook/animated-computed.md

```diff
@@ -73,7 +73,7 @@ turns it into a UDim2 position for The Thing to use. You can imagine this as the
 teleport around.
 
 ```Lua
-			scope:Computed(function(use)
+			scope:Computed(function(use, scope)
```

Comment:
Scope shouldn't be provided as it is never used

Reply:
```suggestion
			scope:Computed(function(use)
```

Reply:
reverted